dkist/net/tests/test_attr_walker.py

```diff
@@ -20,6 +22,41 @@ def query_or_instrument():
     """
     return (a.Instrument("VBI") | a.Instrument("VISP")) & a.Time("2020/06/01", "2020/06/02")
 
+@pytest.fixture(scope="function")
+def boundingbox_params():
+    """
+    Create possible bounding box input coordinates and args
+    for inputs to the bounding box tests.
+    """
+    bottom_left_icrs = SkyCoord([ICRS(ra=1 * u.deg, dec=2 * u.deg, distance=150000000 * u.km)],
+                                  obstime='2021-01-02T12:34:56')
+    top_right_icrs = SkyCoord([ICRS(ra=3 * u.deg, dec=4 * u.deg, distance=150000000 * u.km)],
+                                obstime='2021-01-02T12:34:56')
+    bottom_left_vector_icrs = SkyCoord([ICRS(ra=1 * u.deg, dec=2 * u.deg, distance=150000000 * u.km),
+                                   ICRS(ra=3 * u.deg, dec=4 * u.deg, distance=150000000 * u.km)],
+                                  obstime='2021-01-02T12:34:56')
```

Comment:
small nitpick, can you clean up the indentation here so things line up with the bracket they are enclosed in?

Reply:
Yep! The 'obstime' kwarg belongs to 'SkyCoord'. Hopefully this next push clears that up.